istantest.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
haha no. `istantestd` is the new version of `istantest` because @awlayton pointed out somewhere above that in `istantest` I am calling `instrument` and then call `cover` which causes double instrumentation and confuses coveralls. I didn't want to delete old script, but we could if you want to get rid of it.